src/TreeHouse/QueueBundle/DependencyInjection/TreeHouseQueueExtension.php

```diff
@@ -42,11 +46,11 @@ public function load(array $configs, ContainerBuilder $container)
      * @param array            $config
      * @param ContainerBuilder $container
      */
-    protected function loadDriver(array $config, ContainerBuilder $container)
+    private function loadDriver(array $config, ContainerBuilder $container)
     {
         $container->setParameter('tree_house.queue.driver', $config['driver']);
         $container->setAlias(
-            'tree_house.queue.factory',
+            'tree_house.amqp.factory',
```

Comment:
Shouldn't we try to have every service in this bundle prefixed with `tree_house.queue...`? 

With this format it almost seems like we have a bundle named `TreeHouseAmqpBundle`.